toolkit/tools/imagegen/configuration/partition_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+package configuration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//TestMain found in configuration_test.go.
+
+var (
+	validPartition Partition = Partition{
+		FsType: "ext4",
+		ID:     "MyPartID",
+		Name:   "MyPartitionName",
+		End:    0,
+		Start:  100,
+		Flags:  []PartitionFlag{"dmroot"},
```

Comment:
>"dmroot" [](start = 26, length = 8)

Nit: why not use the constant?

Reply:
Agreed here, constant is fine. We are testing the Partition struct rather than the Flag struct.
Updated.